prog/vnet/load_balancer_health_probes.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class Prog::Vnet::LoadBalancerHealthProbes < Prog::Base
+  subject_is :load_balancer
+
+  def vm
+    @vm ||= Vm[frame.fetch("vm_id")]
+  end
+
+  label def health_probe
+    response_code = nil
+    begin
+      endpoint = "#{vm.nics.first.private_ipv4.network}:#{load_balancer.dst_port}#{load_balancer.health_check_endpoint}"
```

Comment:
ditto on nics.first. Maybe should wrap them in something that complains.

Reply:
answered in the other thread.